cli/internal/util/run_opts.go

```diff
@@ -27,6 +27,8 @@ type RunOpts struct {
 	Parallel bool
 
 	EnvMode EnvMode
+	// Whether or not to perform framework inference.
```

Comment:
nit, not blocking (take it or leave it)
```suggestion
	// Whether or not to infer the framework for each workspace.
```